packages/openapi-spec-builder/src/openapi-spec-builder.ts

```diff
@@ -120,7 +120,7 @@ export class OpenApiSpecBuilder extends BuilderBase<OpenApiSpec> {
 export class OperationSpecBuilder extends BuilderBase<OperationObject> {
   constructor() {
     super({
-      responses: {},
+      responses: {'200': {description: ''}},
```

Comment:
Let's use a non-empty string please. For example `description: 'An undocumented response body.'`

Reply:
PTAL, this comment hasn't been addressed yet.

> Let's use a non-empty string please. For example description: 'An undocumented response body.'